katas/content/distinguishing_states/index.md

```diff
@@ -0,0 +1,48 @@
+# Distinguishing Quantum States
+
+@[section]({
+    "id": "distinguishing_states__overview",
+    "title": "Overview"
+})
+
+This kata is designed to get you familiar with the concept of measurements and using them for analyzing quantum states and with Q# programming.
+
+**This kata covers the following topics:**
+
+- single-qubit measurements,
+- distinguishing orthogonal and nonorthogonal states.
+
+**What you should know to start working on this kata:**
+
+- Dirac notation for single-qubit and multi-qubit quantum systems
+- Basic single-qubit and multi-qubit gates
+- Single-qubit quantum measurements and their effect on quantum systems
+
+@[section]({
+    "id": "distinguishing_states__orthogonal_states",
+    "title": "Distinguishing Orthogonal States"
+})
+
+@[exercise]({
+    "id": "distinguishing_states__zero_one",
+    "title": "|0〉 or |1〉?",
+    "path": "./zero_one/",
+    "qsDependencies": [
+        "../KatasLibrary.qs"
+    ]
+})
+
+
+@[section]({
+    "id": "distinguishing_states__nonorthogonal_states",
+    "title": "Distinguishing Nonorthogonal States"
+})
+
```

Comment:
Should a TODO be added here as well since this section is empty?

Reply:
It's implicit :-) It should be enough to have one TODO in the end of the kata to remind us that it's incomplete